File: src/main/java/de/itsTyrion/flusenbot/command/commands/Ping.java
package de.itsTyrion.flusenbot.command.commands;

import com.pengrad.telegrambot.model.Chat;
import com.pengrad.telegrambot.model.User;
import de.itsTyrion.flusenbot.command.Command;
import de.itsTyrion.flusenbot.command.CommandInfo;

@CommandInfo(name = "ping")
public class Ping extends Command {

    @Override
    protected boolean execute(String[] args, Chat chat, User user) {
        long now = System.currentTimeMillis();
        int id = reply(chat, "Pong!").message().messageId();
        editMessage(chat, id, "Pong! (" + (System.currentTimeMillis() - now) + "ms)");
        return true;
    }
}

File: src/main/java/de/itsTyrion/flusenbot/command/commands/Ping.kt
package de.itsTyrion.flusenbot.command.commands

import com.pengrad.telegrambot.model.Message
import de.itsTyrion.flusenbot.command.Command

class Ping : Command("ping") {

    override fun execute(args: Array<String>, msg: Message): Boolean {
        val now = System.currentTimeMillis()
        val id = reply(msg.chat(), "Pong!").message().messageId()
        editMessage(msg.chat(), id, "Pong! (" + (System.currentTimeMillis() - now) + "ms)")
        return true
    }
}